tests/unit/commands/_utils/test_options.py

```diff
@@ -83,6 +83,15 @@ def test_verify_ctx(self, get_template_data_mock, os_mock):
         self.assertEqual(ctx.template_dict, "dummy_template_dict")
         os_mock.path.abspath.assert_called_with(expected)
 
+    def test_verify_ctx_template_file_param(self):
```

Comment:
can we add a specific test for this in test_samconfig.py file

Reply:
sure, new test case is added in that file